LogicalTypes.md

```diff
@@ -684,44 +702,68 @@ optional group my_list (LIST) {
 }
 ```
 
-Some existing data does not include the inner element layer. For
-backward-compatibility, the type of elements in `LIST`-annotated structures
-should always be determined by the following rules:
+##### 2-level structure
+
+Some existing data does not include the inner element layer, resulting in a
+`LIST` that annotates a 2-level structure. Unlike the 3-level structure, the
+repetition of a 2-level structure can be `optional`, `required`, or `repeated`.
```

Comment:
"the repetition of a 2-level structure can be" so here means the outer "list-repetition"? Should we denote it as out-most level?

Reply:
I think this is clear because we are talking about the whole 2-level structure here.